fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -2423,8 +2423,7 @@ public List<ResultRow> executeInternalQuery() {
                     analyze(context.getSessionVariable().toThrift());
                 }
             } catch (Exception e) {
-                LOG.warn("Internal SQL execution failed, SQL: {}", originStmt, e);
-                return resultRows;
+                throw new RuntimeException("Failed to execute internal SQL", e);
```

Comment:
do we need to add origin sql into exception msg?

Reply:
I think that's unneccessary